### PR TITLE
Formalize support policy, start README reorg.

### LINKED ISSUE #1879: Formalize, publish support policy
**[Mailing List](https://groups.google.com/forum/#!forum/liftweb) thread:** https://groups.google.com/d/msg/liftweb/POl2WS64-SU/mQPFUs4yAwAJ

Let's formalize and publish our support policy of Lift versions per the thread above.

### PR DESCRIPTION
**Pull request by farmdawgnation**
This PR closes #1879.

This bled into #1875 a bit, but I wouldn't say this totally finishes that ticket.

Per the [ML discussion](https://groups.google.com/d/msg/liftweb/POl2WS64-SU/mQPFUs4yAwAJ) this formalizes and publishes our support policy in the GitHub repository. I also took the liberty of making some improvements to the flow of the README, but it's still very much a work in progress. Wanted to go ahead and open this up and have it considered for merge in its current state, however.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -18,14 +20,17 @@ The policy on issues is:
...
+* The contributor's signature is already in or added to /contributors.md file.
 
 Note that only Lift committers can merge a pull request, and accepting or rejecting
 a pull request is entirely at the discretion of the Lift committers. By merging a
 pull request, the Lift committers are taking responsibility for maintaining that code,
 and there are many reasons why this might not be desirable for certain features. We
 will strive to communicate clearly why pull requests are not accepted if we decide
 not to accept them.
+
+[supfile]: https://github.com/lift/framework/blob/master/SUPPORT.md
```
Comment (Shadowfiend) [inline CONTRIBUTING.md]: sup gif

File: SUPPORT.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+Because maintenance of the Framework is a  volunteer effort and all of the Committers have busy
+lives with precious free time, they must be intentional with the community support they provide for
+Lift. To that end, the Committers have decided to embrace the following support policy for releases
+of Lift:
+
+* Lift ships a new release every six months, with three milestones and at least one Release
+Candidate building up to each release. Each normal release results in a bump of the middle number
+of the version (e.g. 3.1.0 to 3.2.0).
+* Brand new features are always a part of new releases. Breaking changes will only ship as a part
```
Comment (Shadowfiend) [inline SUPPORT.md]: Perhaps ‘API-incompatible’ changes for specificity?

File: SUPPORT.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+of a major release, which results in a bump of the first number of the version (e.g. 2.6.0 to 3.0.0).
+* Bug fixes and minor performance improvements will be shipped for releases for 18 months after
```
Comment (Shadowfiend) [inline SUPPORT.md]: I would change this to “can, by consensus of the Committers, be shipped…” Feels like “will” is a bit strong here…

File: SUPPORT.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+* Opening a [Mailing List][ml] thread asking for help from the committers.
+* Filing an issue in GitHub.
+* Opening a Pull Request intended to patch an older version of Lift.
+
+## Common issues
+
+We're sure that this support policy won't meet every organization's needs. To that end, the
+committers wanted to address a few specific problems that may come up as a result of this support
+strategy.
```
Comment (Shadowfiend) [inline SUPPORT.md]: This should probably be followed by a colon.

**Comment by Shadowfiend** [comment]
Left a few minor remarks but this is looking excellent.

**Comment by farmdawgnation** [comment]
Made revisions as requested.

File: README.md [hunk 5]
```diff
@@ -79,37 +61,63 @@ Or, you can add Lift to your `pom.xml` like so:
...
 ### Other Repostories
 
 There are a variety of other repositories available on the Lift GitHub page. While many are concerned with building Lift or are build program archetypes, there are two you will probably encounter fairly frequently as a Lift user:
 
 #### modules
 
-The [modules](https://github.com/liftmodules) repository contains the many add-on modules that are part of the Lift project. If you don't find a module you need here, consider [creating a module](http://www.assembla.com/spaces/liftweb/wiki/Modules) and sharing it with the community.
-
-Please note that the modules project does accept pull requests.
+The [modules](https://github.com/liftmodules) organization contains the many add-on modules that are
```
Comment (karma4u101) [inline README.md]: ... contains some of the many Lift add-on modules ...

File: README.md [hunk 6]
```diff
@@ -79,37 +61,63 @@ Or, you can add Lift to your `pom.xml` like so:
...
+part of the Lift project. If you don't find a module you need here, consider
```
Comment (karma4u101) [inline README.md]: ... If you don't find a module you need here see also [https://liftweb.net/lift_modules](https://liftweb.net/lift_modules) or consider ...

File: README.md (hunk 5 above)
Comment (karma4u101) [inline README.md]: ... no biggie but probably more accurate.

File: README.md (hunk 6 above)
Comment (karma4u101) [inline README.md]: ... also no big deal, but the list on lifts homepage gives a good overview. 

Overall, this looks good to me +1

File: README.md [hunk 7]
```diff
@@ -79,37 +61,63 @@ Or, you can add Lift to your `pom.xml` like so:
...
-This component includes all of Lift's core HTTP and web handling. Including `lift-webkit` in your build process should be sufficient for basic applications and will include `lift-core` as a transitive dependency.
+### This Repository
 
-#### persistence
+This repository, `framework`, contains the following components:
 
-This component includes Mapper and Record, Lift's two ORMs. While you needn't use either and can use the ORM of your choice, Mapper and Record integrate nicely with Lift's idioms. Mapper is an ORM for relational databases, while Record is a broader ORM with support for both SQL databases and NoSQL datastores.
+* **core:** Core elements used by Lift projects. If you wish to reuse some of Lift's helpers and
+constructs, such as `Box`, this component may be all you need. However, a web application will most
+likely require one or more of Lift's components.
```
Comment (EgoWumpus) [inline README.md]: This sentence seems a bit tautological. Do you mean "require one or more of Lift's *other components"?

Comment (farmdawgnation) [inline README.md]: Indeed I do.

**Comment by farmdawgnation** [comment]
I've addressed all concerns. I think it's time to ship it.

**Pull request merged**